tests/checkpointing/test_model_checkpoint.py

```diff
@@ -178,48 +178,54 @@ def test_model_checkpoint_format_checkpoint_name(tmpdir):
     # empty filename:
     ckpt_name = ModelCheckpoint._format_checkpoint_name('', 3, {})
     assert ckpt_name == 'epoch=3'
+
     ckpt_name = ModelCheckpoint._format_checkpoint_name(None, 3, {}, prefix='test')
     assert ckpt_name == 'test-epoch=3'
+
     # no groups case:
     ckpt_name = ModelCheckpoint._format_checkpoint_name('ckpt', 3, {}, prefix='test')
     assert ckpt_name == 'test-ckpt'
+
     # no prefix
     ckpt_name = ModelCheckpoint._format_checkpoint_name('{epoch:03d}-{acc}', 3, {'acc': 0.03})
     assert ckpt_name == 'epoch=003-acc=0.03'
+
     # prefix
     char_org = ModelCheckpoint.CHECKPOINT_JOIN_CHAR
     ModelCheckpoint.CHECKPOINT_JOIN_CHAR = '@'
     ckpt_name = ModelCheckpoint._format_checkpoint_name('{epoch},{acc:.5f}', 3, {'acc': 0.03}, prefix='test')
     assert ckpt_name == 'test@epoch=3,acc=0.03000'
     ModelCheckpoint.CHECKPOINT_JOIN_CHAR = char_org
-    # no filepath set
-    ckpt_name = ModelCheckpoint(monitor='early_stop_on', filepath=None).format_checkpoint_name(3, {})
+
+    # no dirpath set
+    ckpt_name = ModelCheckpoint(monitor='early_stop_on', dirpath=None).format_checkpoint_name(3, {})
     assert ckpt_name == 'epoch=3.ckpt'
-    ckpt_name = ModelCheckpoint(monitor='early_stop_on', filepath='').format_checkpoint_name(5, {})
+    ckpt_name = ModelCheckpoint(monitor='early_stop_on', dirpath='').format_checkpoint_name(5, {})
     assert ckpt_name == 'epoch=5.ckpt'
+
     # CWD
-    ckpt_name = ModelCheckpoint(monitor='early_stop_on', filepath='.').format_checkpoint_name(3, {})
-    assert Path(ckpt_name) == Path('.') / 'epoch=3.ckpt'
-    # dir does not exist so it is used as filename
-    filepath = tmpdir / 'dir'
-    ckpt_name = ModelCheckpoint(monitor='early_stop_on', filepath=filepath, prefix='test').format_checkpoint_name(3, {})
-    assert ckpt_name == tmpdir / 'test-dir.ckpt'
-    # now, dir exists
-    os.mkdir(filepath)
-    ckpt_name = ModelCheckpoint(monitor='early_stop_on', filepath=filepath, prefix='test').format_checkpoint_name(3, {})
-    assert ckpt_name == filepath / 'test-epoch=3.ckpt'
+    ckpt_name = ModelCheckpoint(monitor='early_stop_on', dirpath='.').format_checkpoint_name(3, {})
+    assert ckpt_name == os.path.join(os.path.realpath(Path('.')), 'epoch=3.ckpt')
+
     # with ver
-    ckpt_name = ModelCheckpoint(monitor='early_stop_on',
-                                filepath=tmpdir / 'name', prefix='test').format_checkpoint_name(3, {}, ver=3)
+    ckpt_name = ModelCheckpoint(
+        monitor='early_stop_on', dirpath=tmpdir, filename='name', prefix='test'
+    ).format_checkpoint_name(4, {}, ver=3)
     assert ckpt_name == tmpdir / 'test-name-v3.ckpt'
 
+    # with ver
```

Comment:
This comment is not correct

```suggestion
    # using slashes
```